packages/multi-select-combo-box/src/vaadin-multi-select-combo-box-internal.js

```diff
@@ -81,6 +81,18 @@ class MultiSelectComboBoxInternal extends ComboBoxDataProviderMixin(ComboBoxMixi
     this._toggleElement = this.querySelector('.toggle-button');
   }
 
+  /**
+   * @protected
+   * @override
+   */
```

Comment:
**nit:** Would suggest clarifying the origin of the method as it took me a while to go down the chain of mixins to find out where the method comes from. 

```suggestion
  /**
   * Override method from `InputMixin`.
   *
   * @protected
   * @override
   */
```